### PR TITLE
Feature/rifex 74/refactor design

### PR DESCRIPTION
**Pull request by JONAF2103**
This introduces the refactor for the design changes that we need, we have several changes here that affects the ui and reallocates some components in the screen, also we removed the old tabs component for a new component made by us since nifty is not compatible with any lib found about that.
Here some screenshots:
imagen
imagen
imagen
imagen

### CODE REVIEW

**Review by marcosmartinez7 — APPROVED** [review]
lgtm

File: old-ui/app/components/account-dropdowns/account-dropdowns.component.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ class AccountDropdowns extends Component {
   }
 
   static propTypes = {
-    identities: PropTypes.objectOf(PropTypes.object),
+    identities: PropTypes .objectOf(PropTypes.object),
```
Comment (rorolopetegui) [inline old-ui/app/components/account-dropdowns/account-dropdowns.component.js]: Can you correct this space?

File: package.json [hunk 2]
```diff
@@ -190,7 +192,6 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.0.0",
     "react-simple-file-input": "^2.0.0",
-    "react-tabs": "^3.1.0",
```
Comment (rorolopetegui) [inline package.json:193]: Why are we removing this dependency?

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
It lgtm, i just required a minor change

File: package.json (hunk 2 above)
Comment (JONAF2103) [reply package.json:193]: I have explained that on the commit comment, we have our own tabs component now, this one was incompatible.

**Review by rorolopetegui — APPROVED** [review]
lgtm now

**Pull request merged**